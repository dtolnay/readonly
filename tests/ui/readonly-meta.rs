#[readonly::make]
pub struct S {
    #[readonly = "..."]
    namevalue: i32,

    #[readonly(...)]
    list: i32,
}

fn main() {}
