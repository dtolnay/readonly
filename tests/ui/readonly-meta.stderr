error: unexpected token in attribute
 --> tests/ui/readonly-meta.rs:3:16
  |
3 |     #[readonly = "..."]
  |                ^

error: unexpected token in attribute
 --> tests/ui/readonly-meta.rs:6:15
  |
6 |     #[readonly(...)]
  |               ^
